### ISSUE TITLE
OSMC 2018-06.1Vero 4k: script.module.pycryptodome version 3.4.3 could not be satisfied 

### ISSUE THREAD

**Issue by marekzukal** [asciidisco/plugin.video.netflix#406]
*I'm submitting a ...*
  - [X] bug report
  - [ ] feature request
  - [ ] support request

## General infomration
Hello, I have  OSMC 2018-06.1Vero 4k  and I never succeeded installing this plugin. First tried in January and still no luck.

16:08:08.353 T:3589354240   ERROR: CAddonInstallJob[plugin.video.netflix]: The dependency on script.module.pycryptodome version 3.4.3 could not be satisfied.

pip list tells me that pycryptodomex (3.4.3) is installed. This is an older version that the plugin wants but I tried it with the latest one pycryptodomex (3.6.1) and the results were the same.

Help will be much appreciated. Thanks.

**Comment by MilhouseVH** [comment]
> Having same issue.
plugin.video.netflix v0.13.8
Latest Libreelec 9.0 git build 2018-06-16
script.module.pycryptodome - Unavailble

In the case of LibreELEC (I can't say for OSMC) this is a Kodi error/bug/issue - script.module.pycryptodome is built-in to the LibreELEC image and isn't provided as an online download which is where Kodi expects to find it, consequently Kodi reports it as unavailable. Just ignore the "unavailable" message for built-in modules such as script.module.pycryptodome as the message from Kodi is *wrong* and plugin.video.netflix will load script.module.pycryptodome without a problem when you run the add-on.

> /usr/share/kodi/addons/script.module.pycryptodome/addon.xml: (version="3.4.3")
/usr/lib/python2.7/site-packages/pycryptodome-3.6.1-py2.7.egg-info/(5 files)
Notice that the version number differ....?

The version details [come from Kodi](https://github.com/xbmc/xbmc/blob/master/addons/script.module.pycryptodome/addon.xml) and are out of date as LibreELEC now ships the latest version of the library. It's not ideal, but not a problem either - it's entirely cosmetic.

> Guess that above is the reason why it is not possible to log in.

No, it's not. And it's not possible to tell what your actual problem is from your log as you've posted a snippet rather than a complete log, and you haven't enabled debug logging.

I would suggest you try re-installing plugin.video.netflix (from the Kodinerds repository, so you get the inputstream.helper, then update from github) and if the problem continues then open a new issue, because this issue is not relevant to LibreELEC.

**Comment by liberty-developer** [comment]
@Phil67ago pls. provide full logs and enable debug logging in settings (after enabling pls. restart kodi)

**Comment by Papale63** [comment]
Hello,
with OSMC this shown depency is not needed, just grab the zip from github, extract it, find addon.xml, open it, remove the line ( near at top ) containing "3.4.3" ;-), save the file, zip the content of the folder plugin.video.netflix-master again to have a new zip - in OSMC install this zip ;-) ...

**Comment by Phil67ago** [comment]
Login works now after update to 0.13.10!!
/Thanks
Well done!!!!

**Comment by Papale63** [comment]
It was a pleasure to me ;-) 
--
Sent from myMail for Android Monday, 18 June 2018, 03:08PM +02:00 from Phil67ago  notifications@github.com :

**Comment by marekzukal** [comment]
OK, removing the dependency from the zip file worked. Thanks for the advice. Even though there could be a mention somewhere how to do this if this does not work automatically. I did suspect that there is no Kodi package called cryptodome and that since I installed it manually, there is no way to force the dependency as a package. But I did not figure out how to bypass it.
So to sum up:
I can install now
I can login and browse.
I have some video playing issues. I will try to investigate and maybe start another thread.
Thanks
